seo4ajax.go

```diff
@@ -16,6 +16,7 @@ import (
 
 	"github.com/cenkalti/backoff"
 	"github.com/go-kit/kit/log"
+	"github.com/go-kit/kit/log/level"
```

Comment:
is it really necessary to add a new dependency for one log line call? From our perspective it may makes sense, but from the open source community it looks weird to introduce this new dependency, that we use that package might be a special case from our stack. Maybe adding this in a later PR for every call of the logger makes more sense?

Reply:
Yes, we can do that. We were using the go-kit logger before, so I didn't think much about adding this package, but we can do another (proper) PR for the logger.